tests/integration/tier0/log-command/test_log_command.py

```diff
@@ -0,0 +1,40 @@
+C2R_LOG = "/var/log/convert2rhel/convert2rhel.log"
+
+
+def test_verify_logfile_starts_with_command(shell):
+    """
+    This test verifies, that the command passed to the command line is at the beginning of the log file.
+    Also verify, that the passed password is obfuscated.
+    """
+    serverurl = "subscription.pls.register.me"
+    username = "mr-anderson"
+    password = "redpill"
```

Comment:
Just a small tip. I would change this to something completely different as this term, nowadays, is highly associated with bad things (at least, in Brazil we are very used to see this term being used with another interpretation). 

Although, it's clear here that it is a reference to the Matrix film, I still would change this to not leave any margins for bad interpretations. 

(Probably I'm being a bit paranoid, I know).

Reply:
It's used within a lot of controversial groups in USA as well, I'll change it